src/baldr/datetime.cc

```diff
@@ -442,6 +442,13 @@ bool is_restricted(const bool type,
   return (dow_in_range && dt_in_range);
 }
 
+uint32_t second_of_week(uint32_t epoch_time, const NodeInfo* node, double elapsedtime) {
```

Comment:
when tracking time within FormPath we need to account for changes in timezone which may change the second of the week that we are on (which changes which speed we get in the edge costing). we add a small helper method to allow us to always get the timezone offset seconds of week after offseting from an epoch time

Reply:
Could you add this comment as a docstring for the function?

Reply:
it is in the header we dont usually copy those to the impl